lighthouse-core/report/html/report-styles.css

```diff
@@ -341,7 +341,8 @@
 .lh-audit-group a,
 .lh-category-header__description a,
 .lh-audit__description a,
-.lh-footer a {
+.lh-footer a 
```

Comment:
this seems like it is missing a `,`?

Reply:
Thanks, I guess haste is risky, even for a one-liner.
Sorry for that and thanks for the review :)